Changes

```diff
@@ -1,4 +1,17 @@
-Release history for Perl module Zonemaster::CLI
+Release history for Zonemaster Component Zonemaster-CLI 
+(Perl module Zonemaster::CLI).
```

Comment:
This changelog is not limited to just that module. The Zonemaster-CLI distribution would be more accurate. Alternatively, I believe this line could be removed without loss of information.